### ISSUE TITLE
Add rate limiter or fail2ban on /api/login

### ISSUE THREAD

**Issue by noerw** [SpeckiJ/UserManager#10]
otherwise LDAP admin is just one bruteforce away

readily available middleware:
https://github.com/didip/tollbooth_httprouter
https://github.com/didip/tollbooth

**Comment by noerw** [comment]
implemented in https://github.com/fs-geofs/UserManager/commit/9c3966834a0d33a9762054a0374e0fd8874b31a5

**Comment by christophfriedrich** [comment]
@noerw wrote:
> implemented in fs-geofs@9c39668

Then this issue can be closed?

**Comment by noerw** [comment]
idk, depends on how we want to track issues and the code on this project.
The code in this repo doesn't include this change (yet).
However `master` from fs-geofs is currently deployed. 

Proposal to clean this up:
- [x] I submit a pull from fs-geofs to SpeckiJ (#11)
- [ ] ~we transfer https://github.com/fs-geofs/UserManager/pull/1 to this repo (not sure if possible. just reopen here?)~
- [x] Once merged we delete https://github.com/fs-geofs/UserManager
- [ ] @SpeckiJ transfers repo ownership to fs-geofs

**Comment by SpeckiJ** [comment]
@noerw As I would like to keep this repository in my personal GitHub Repository, I would propose this:

- All development will take place on [fs-geofs/UserManager](https://github.com/fs-geofs/UserManager)
- All deployment will be based on [fs-geofs/UserManager](https://github.com/fs-geofs/UserManager)
- We transfer all Issues to fs-geofs (or just close them by implementing them)
- I can keep my branch up-to-date by doing PR's from the [fs-geofs/UserManager](https://github.com/fs-geofs/UserManager) Repository

Which would lead to the following tasks:

- [x] fork [speckij/UserManager](https://github.com/speckij/UserManager) to fs-geofs
- [x] Transfer the 3 remaining Issues to [fs-geofs/UserManager](https://github.com/fs-geofs/UserManager)

**Comment by noerw** [comment]
lol, I just deleted the fork :D 
but i'm fine with either way

**Comment by christophfriedrich** [comment]
I don't fully understand what you guys now did to the repos, but what I see is a lot of notifications and a confusing state of this project's issues 👎

**Comment by christophfriedrich** [comment]
Why do we now have two issue trackers?!

**Comment by SpeckiJ** [comment]
@christophfriedrich:

The repository status is described here: https://github.com/SpeckiJ/UserManager/issues/10#issuecomment-487164339

tldr: fs-geofs will be the active fork that is actually used/maintained/developed against
